### ISSUE TITLE
[content switcher] add size variants

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-design-kit#367]
Add a 32px and 48px size content switcher to the kits.

- [x] In code ([Merged PR](https://github.com/carbon-design-system/carbon/pull/6946))
- [x] Add to kits
   - [x] White theme 
   - [x] Gray 10 theme 
   - [x] Gray 90 theme 
   - [x] Gray 100 theme (in progress)

**Comment by aagonzales** [comment]
Adding to this issue since its related and can be done at the same time.

We're missing a `selected + focus` state
![image](https://user-images.githubusercontent.com/11670886/97922793-a3416e80-1d22-11eb-858c-c7064bd12b2e.png)

**Comment by dianatran18** [comment]
Hey @laurenmrice I realized I can't assign myself to the issue, but I have 3/4 themes built out I'll get the last one built out this week and then send these over.

I can't remember how you guys handle adding in/updating symbols in the Carbon kits, so I might need help with that, or I can just pass the files over to one of you guys if you wanted to add it in so I don't mess anything up! I built them in a local copy of the Carbon kits.
